plugins/modules/powervm_lpar_instance.py

```diff
@@ -19,7 +19,7 @@
     - Anil Vijayan (@AnilVijayan)
 short_description: Create/Delete an AIX/Linux or IBMi partition
 notes:
-    - Currently supports creation of partition (powervm instance) with only processor and memory settings
+    - Currently supports creation of partition (powervm instance) with only processor and dedicated memory settings
```

Comment:
should like "Currently supports creation of partition (powervm instance) with only processor and memory settings on dedicated mode"

Reply:
Done